packages/select/src/Select/components/TriggerButton/TriggerButton.module.scss

```diff
@@ -121,6 +121,7 @@
 }
 
 .reversed.disabled {
+  background-color: transparent;
```

Comment:
Should this be transparent or some other explicit colour instead?

Reply:
As far as I could tell, it is currently transparent as the reversed, disabled variant - had to set it explicitly here, because the CSS spilled over.

Reply:
I suppose if that's what it's meant to be 😬